test/integration/targets/k8s/runme.sh

```diff
@@ -12,14 +12,26 @@ MYTMPDIR=$(mktemp -d 2>/dev/null || mktemp -d -t 'mytmpdir')
 # but for the python3 tests we need virtualenv to use python3
 PYTHON=${ANSIBLE_TEST_PYTHON_INTERPRETER:-python}
 
+# Test graceful failure for missing kubernetes-validate
+virtualenv --system-site-packages --python "${PYTHON}" "${MYTMPDIR}/openshift-validate-not-installed"
+source "${MYTMPDIR}/openshift-validate-not-installed/bin/activate"
+$PYTHON -m pip install git+https://github.com/willthames/openshift-restclient-python@kubernetes-validate
+ansible-playbook -v playbooks/validate_not_installed.yml "$@"
+
+# Test graceful failure for missing kubernetes-validate
+virtualenv --system-site-packages --python "${PYTHON}" "${MYTMPDIR}/openshift-validate-installed"
+source "${MYTMPDIR}/openshift-validate-installed/bin/activate"
+$PYTHON -m pip install git+https://github.com/willthames/openshift-restclient-python@kubernetes-validate kubernetes-validate
+ansible-playbook -v playbooks/validate_installed.yml "$@"
+
 # Test graceful failure for older versions of openshift
 virtualenv --system-site-packages --python "${PYTHON}" "${MYTMPDIR}/openshift-0.6.0"
 source "${MYTMPDIR}/openshift-0.6.0/bin/activate"
 $PYTHON -m pip install 'openshift==0.6.0' 'kubernetes==6.0.0'
-ansible-playbook -v playbooks/merge_type_fail.yml "$@"
+ansible-playbook -v playbooks/older_openshift_fail.yml "$@"
 
 # Run full test suite
 virtualenv --system-site-packages --python "${PYTHON}" "${MYTMPDIR}/openshift-recent"
 source "${MYTMPDIR}/openshift-recent/bin/activate"
-$PYTHON -m pip install 'openshift>=0.7.0'
+$PYTHON -m pip install 'git+https://github.com/willthames/openshift-restclient-python@append_hash'
```

Comment:
Why are these pip installs done from GH?

Reply:
@mattclay this is a WIP PR and those are WIP versions of the modules - once those modules are released I'll update this PR to use the proper pypi versions

Reply:
Thanks for the clarification.